### DISCUSSION TITLE
Just a quick question

### DISCUSSION THREAD

**Discussion by ChillVibesMushroom** [ssh-mitm/ssh-mitm#113] [General]
yo, everything works great thanks alright uh what I want to know is how would I go about spoofing my ipv4 and being able to connect with ssh-mitm.

i run sudo ifconfig wlan0 my desired phone ipv4  and use termux on another device to connect via ssh desired ipv4 -p 10022 and I'm not able to connect ssh just hangs.

:{

**Comment by manfred-kaiser** [comment]
SSH-MITM supports a transparent mode. Please keep in mind transparent mode is experimental and requires root privileges for SSH-MITM to run.

The docs describe some example firewall rules which are needed to setup the proxy: https://docs.ssh-mitm.at/advanced-usage.html

**Option 1:**
To redirect the traffic to SSH-MITM you can configure SSH-MITM's server as the default gateway on the client. 


**Option 2:**
If you are not able to configure the default gateway you can try arp poisoning. arp poisoning is not integrated in SSH-MITM, so you have to use another program like "arpspoof" from the "dsniff" package (Debian/Ubuntu) or ettercap.

- Tutorial for arpspoof: https://tutorialedge.net/security/arp-spoofing-for-mitm-attack-tutorial/
- Tutorial for ettercap: https://www.tutorialspoint.com/ethical_hacking/ethical_hacking_arp_poisoning.htm


**Please contribute :wink:**
I would be glad if you can update the documentation, if your setup works, to help other users: https://github.com/ssh-mitm/ssh-mitm/blob/master/doc/advanced-usage.rst
